package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -170,7 +170,26 @@ append_address() {
 }
 
 append_ipset() {
-	xappend "--ipset=$1"
+	if [ -x /sbin/fw4 ]; then
```

Comment:
The check for fw4 shouldn't be required. We either have iptables or nftables. If nftables is present, dnsmasq fails silently to add the ipsets. Means, this function can append the ntfset and ipset parameter at the same time.

You can find such an implementation in my staging tree.